CDTDatastore/HTTP/CDTSessionCookieInterceptor.m

```diff
@@ -77,21 +67,6 @@ - (CDTHTTPInterceptorContext *)interceptRequestInContext:(CDTHTTPInterceptorCont
 }
 
 /**
- We assume a 401 means that the cookie we applied at request time was rejected. Therefore
- clear it and tell the HTTP mechanism to retry the request. For all other responses, there's
- nothing for this interceptor to do.
- */
-- (CDTHTTPInterceptorContext *)interceptResponseInContext:(CDTHTTPInterceptorContext *)context
```

Comment:
Correct this case was never handled for the _session endpoint, do you think it falls under the scope of this PR?